erpnext/accounts/doctype/unreconcile_payment/unreconcile_payment.py

```
@@ -149,19 +148,44 @@ def get_linked_payments_for_doc(
 			res = query.run(as_dict=True)
+
+			if _dt == "Payment Entry":
+				# Add advance payments linked to this payment entry
+				res += get_advance_linked_payments_for_doc(doctype, docname)
+
 			return res
+
 	return []
 
 
+def get_advance_linked_payments_for_doc(doctype: str | None = None, docname: str | None = None):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)